tools/Dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM        ocaml/opam2:debian-9-ocaml-4.07
+FROM        ocaml/opam2:debian-10-ocaml-4.07
 MAINTAINER  Christian Lindig <christian.lindig@citrix.com>
 
 COPY --chown=opam:opam . /tmp/xs-opam
 
 RUN sudo apt-get update
+RUN sudo apt-get install -qq -yy hwdata libdlm-dev libffi-dev libgmp-dev \
```

Comment:
I am not sure these are strictly necessary because the `opam depext -y xs-toolstack` should install them. We once had problems with that, though.

Reply:
ah you're right - without this line, building the image does work, it just hangs for a while and gives a message which makes it seem like it's going to fail